AoE2ScenarioParser/objects/support/new_effect.py

```diff
@@ -1576,3 +1575,147 @@ def modify_attribute_by_variable(
             variable=variable,
             message=message
         )
+
+    def set_object_cost(
+            self,
+            object_list_unit_id: int | None = None,
```

Comment:
i see various effects also have `item_id` that is usually equals to object or technology value. see that other effects do not set it so i did not set it as well

Reply:
Item ID is indeed ignored (effectively) it's automatically set when writing the scenario, just for parity with the game- it doesn't actually do anything 11